packages/editor/src/components/inserter/menu.js

```diff
@@ -290,7 +291,7 @@ export class InserterMenu extends Component {
 								className="editor-inserter__manage-reusable-blocks"
 								icon="admin-generic"
 								label={ __( 'Manage All Reusable Blocks' ) }
-								href="edit.php?post_type=wp_block"
+								href={ addQueryArgs( 'edit.php', { post_type: 'wp_block' } ) }
```

Comment:
I don't think we should ever reference PHP file inside the `editor` package. I noticed there are other places where we link to WordPress specific url. @youknowriad - should we move out those bits to `edit-post` or make them customizable?

Reply:
There's an issue for that :) #9637

Reply:
Awesome :)